ApiService/Sources/ApiService/MovieService/MovieService.swift

```diff
@@ -0,0 +1,30 @@
+import Combine
+import Foundation
+
+public class MovieService {
+    
+    func getMovies(bySearchTerm search: String) async throws -> [Movie]? {
+        
+        return try await withCheckedThrowingContinuation { continuation in
+            if let url = Bundle.module.url(forResource: search, withExtension: "json") {
```

Comment:
Do we want to hit this with a 'search.lowercased()' to ensure that during our demo Batman and Superman will still be valid search parameters and won't cause any awkward moments?

Reply:
Good idea, I'll fix it.